testsuite/cluster-test/src/main.rs

```diff
@@ -590,16 +593,20 @@ impl ClusterTestRunner {
         info!("Starting suite");
         let suite_started = Instant::now();
         for experiment in suite.experiments {
+            let start_time = Instant::now();
             let experiment_name = format!("{}", experiment);
             let experiment_result = self
                 .run_single_experiment(experiment, None)
                 .await
                 .map_err(move |e| format_err!("Experiment `{}` failed: `{}`", experiment_name, e));
+            let end_time = (Instant::now() - start_time).as_secs() as u64;
             if let Err(e) = experiment_result.as_ref() {
                 self.report.report_text(e.to_string());
                 self.print_report();
                 experiment_result?;
             }
+            self.report
+                .report_text(format!("Time spent {} secs", end_time))
```

Comment:
nit - it would be easier to read the current plain text report in the same line.  Example below

```
10% down : 887 TPS, 4610 ms latency, 6500 ms p99 latency, no expired txns, 419 sec test time 
```

instead of
```
10% down : 887 TPS, 4610 ms latency, 6500 ms p99 latency, no expired txns
Time spent 419 secs
```

Reply:
BTW, do we plan to track and plot the perf results over time?

Reply:
I'm hoping we can just report this info into system that @sunshowers is planning.

Reply:
@sunshowers is there any more details about the plan?

Reply:
changed format to
```
Test runner setup time spent 267 secs
all up : 938 TPS, 4853 ms latency, 5600 ms p99 latency, no expired txns, time spent 270 secs
10% down : 914 TPS, 4470 ms latency, 5400 ms p99 latency, no expired txns, time spent 429 secs
3 Region Simulation : 738 TPS, 6178 ms latency, 7600 ms p99 latency, no expired txns, time spent 267 secs
fixed tps 10 : 10 TPS, 918 ms latency, 1000 ms p99 latency, no expired txns, time spent 248 secs
```